metals/src/main/scala/scala/meta/internal/worksheets/DecorationWorksheetPublisher.scala

```diff
@@ -65,7 +65,11 @@ class DecorationWorksheetPublisher() extends WorksheetPublisher {
       decorations: Array[DecorationOptions]
   ): Unit = {
     val params =
-      new PublishDecorationsParams(path.toURI.toString(), decorations)
+      new PublishDecorationsParams(
+        path.toURI.toString(),
+        decorations,
+        isInline = false
```

Comment:
In the `SyntheticsDecorationProvider` we do a check to see if the client is a `inlineDecorationProvider`, should we do that here as well? For example right now in `nvim-metals` in worksheets I'm getting `isInline: false` being sent here even though it doesn't support inline decorations.